tasks/task_3.py

```diff
@@ -0,0 +1,18 @@
+from selenium import webdriver
+from bs4 import BeautifulSoup
+import pandas as pd
+from utils import scroll_to_page_end, extract_videos_data
+
+url = input("Enter Any Youtube Chanel URL: ")
+url += "/videos"
+driver = webdriver.Chrome(executable_path="chromedriver")
```

Comment:
if you add a space above this line, your code would be more well formatted

Reply:
ok